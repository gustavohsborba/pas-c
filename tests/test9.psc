var

 j, k is int;

 begin
 k := j + k;
 read(j);
 read(k);

 if (k <> 0)
 result := j/k
 else
 result := 0
 end;

 out(result);
 end