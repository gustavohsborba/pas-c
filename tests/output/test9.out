Opening file tests/test9.psc...
ERROR in line 7, column 6: Identifier read not declared in this scope
